deploy/charts/trust-manager/templates/clusterrole.yaml

```
@@ -44,24 +44,24 @@ rules:
   - "events"
   verbs: ["create", "patch"]
 
+{{- if .Values.secretTargets.enabled }}
+{{- if .Values.secretTargets.authorizedSecretsAll }}
 - apiGroups:
   - ""
   resources:
   - "secrets"
-  verbs: ["create", "list", "watch"]
-{{- if .Values.secretTargets.enabled }}
-{{- if .Values.secretTargets.authorizedSecretsAll }}
+  verbs: ["get", "list", "create", "patch", "watch", "delete"]
+{{- else if .Values.secretTargets.authorizedSecrets }}
 - apiGroups:
   - ""
   resources:
   - "secrets"
-  verbs: ["get", "update", "delete", "patch"]
-{{- else if .Values.secretTargets.authorizedSecrets }}
+  verbs: ["list", "watch", "get"]
```

```suggestion
  verbs: ["get", "list", "watch"]
```

Nit: For consistency